node/pkg/node/node.go

```diff
@@ -244,7 +255,7 @@ func GuardianOptionWatchers(watcherConfigs []watchers.WatcherConfig, ibcWatcherC
 
 			for _, wc := range watcherConfigs {
 				if _, ok := watchers[wc.GetNetworkID()]; ok {
-					logger.Fatal("NetworkID already configured", zap.String("network_id", string(wc.GetNetworkID())))
+					return fmt.Errorf("NetworkID already configured: %s", string(wc.GetNetworkID()))
```

Comment:
Minor, but the linter should have caught this. It is idiomatic [to not capitalize the beginning of error strings](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).

Reply:
in this case `NetworkID` is the name of the variable so should be capitalized?

Reply:
Regardless of the variable name this goes against the [official guidance](https://github.com/golang/go/wiki/CodeReviewComments#error-strings) that error strings. It's a small nit, but FYI and a lot of linters will bomb on this.

Reply:
>>Error strings should not be capitalized (*unless beginning with proper nouns or acronyms*)